### PR TITLE
build-scripts: use the git mirror for building Debian and Centos tools

### PR DESCRIPTION
**Pull request by jean-edouard**
OXT-507

Signed-off-by: Jed <lejosnej@ainfosec.com>

### CODE REVIEW

File: build-scripts/debian/build.sh [hunk 1]
```diff
@@ -29,14 +29,15 @@ BRANCH=%BRANCH%
...
 SBUILD="sbuild --purge-deps=never"
 
 # Build
 mkdir $BUILD_DIR
 cd $BUILD_DIR
-git clone -b $BRANCH https://github.com/OpenXT/pv-linux-drivers.git
-git clone -b $BRANCH https://github.com/OpenXT/v4v.git
-git clone -b $BRANCH https://github.com/OpenXT/xctools.git
+git clone -b $BRANCH $GIT_MIRROR/pv-linux-drivers.git
+git clone -b $BRANCH $GIT_MIRROR/v4v.git
```
Comment (rossphilipson) [inline build-scripts/debian/build.sh]: CentOS build should be building libv4v and the switcher too - seems it is missing there?

Comment (jean-edouard) [inline build-scripts/debian/build.sh]: Yes it is.
I'll take care of it in a separate pull request.

**Comment by rossphilipson** [comment]
I am inclined to merge most of these (except the cygwin proxy one) and do a full test of everything. There have been a number of other changes lately too. Merging soon...

**Comment by rossphilipson** [comment]
Tested all the scripts including the new Windows ones. Found a few new issues but mostly it all worked well.

**Pull request merged**